lib/jsonapi-response.js

```diff
@@ -0,0 +1,102 @@
+const attributesMap = require('../fixtures/attributes');
+const relationshipsMap = require('../fixtures/relationships');
```

Comment:
I think these could be named better - "map" suggests they're key => value, whereas they're arrays. Not that I'm suggesting renaming Map => Array since I don't think it's right to name variables after the type of the value they hold!

What about `allAttributes` or `allRelationships`?